### ISSUE TITLE
DeprecationWarning with Pillow==9.1.0

### ISSUE THREAD

**Issue by uri-rodberg** [jazzband/sorl-thumbnail#695]
I tried to use `sorl-thumbnail==12.8.0` with `Pillow==9.1.0` and I receive this error message (with deprecation warnings enabled):

```
DeprecationWarning: ANTIALIAS is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.LANCZOS instead.
```

**Comment by uri-rodberg** [comment]
https://github.com/speedy-net/speedy-net/actions/runs/2214341276

**Comment by uri-rodberg** [comment]
In the meantime, I'm using `Pillow==9.0.1` which doesn't have this DeprecationWarning.

**Comment by claudep** [comment]
Fixed by 8d7bd407be95f3daf

**Comment by uri-rodberg** [comment]
@claudep Can you release a new release with this change, and also supporting Django 4.0 and Python 3.9, 3.10?
